internal/sdkprovider/service/organization/util.go

```diff
@@ -0,0 +1,28 @@
+// Package organization contains the organization related resources and utilities.
+package organization
+
+import (
+	"regexp"
+
+	"github.com/aiven/aiven-go-client"
+)
+
+// regExpOrganizationID is a regular expression that matches organization IDs (they are prefixed with "org").
+var regExpOrganizationID = regexp.MustCompile("^org")
+
+// idToUse is a helper function that returns the ID to use for the API call.
+// If the ID is an organization ID, it will be converted to an account ID.
+// If the ID is an account ID, it will be returned as is.
+func idToUse(client *aiven.Client, id string) (string, error) {
+	if regExpOrganizationID.MatchString(id) {
+		r, err := client.Organizations.Get(id)
+
+		if err != nil {
+			return "", err
+		}
+
+		id = r.AccountID
+	}
+
+	return id, nil
```

Comment:
can it use just [HasPrefix](https://pkg.go.dev/strings#HasPrefix) instead of refexp?